Sources/SwiftInspectorCore/Tests/TypePropertiesSpec.swift

```diff
@@ -84,8 +84,9 @@ final class TypePropertiesSpec: QuickSpec {
       }
 
       context("when the types don't match") {
-        it("fails to merge and asserts") {
-          expect { try type1.merge(with: type3) }.to(throwError())
+        // Disabling this test since it fails in Xcode 12 / Swift 5.3
+        xit("fails to merge and asserts") {
```

Comment:
```suggestion
        it("fails to merge and asserts") {
```

We should make this change ^ before merging.